SimCalorimetry/EcalElectronicsEmulation/bin/GenABIO.cc

```diff
@@ -389,7 +389,7 @@ void fillABTTFFiles(const char ttFlags[nTTInEta][nTTInPhi], ofstream files[]) {
 
 void fillABSRPFiles(const char barrelSrFlags[nBarrelTTInEta][nTTInPhi],
                     const char endcapSrFlags[nEndcaps][nSupercrystalXBins][nSupercrystalYBins],
-                    ofstream files[nAB]) {
+                    ofstream files[]) {
```

Comment:
The fix is technically correct, and let merge it to shut up the clang warning,
However, I wonder if this code is ever used, and how can it work if the argument is not passed by reference...